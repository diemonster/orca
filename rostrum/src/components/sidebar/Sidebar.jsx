import React from 'react';
import './sidebar.css';

function Sidebar() {
  return (
    <div className="sidebar">
      <ul>
        <li><a href="namespace">Namespaces</a></li>
        <li><a href="pod">Pods</a></li>
      </ul>
    </div>
  );
}

export default Sidebar;
