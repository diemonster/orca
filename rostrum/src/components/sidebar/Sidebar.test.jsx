import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Sidebar from './Sidebar';

Enzyme.configure({ adapter: new Adapter() });

describe('components', () => {
  describe('Sidebar', () => {
    it('renders itself', () => {
      const sidebar = shallow(<Sidebar />);
      expect(sidebar).toMatchSnapshot();
    });
  });
});
