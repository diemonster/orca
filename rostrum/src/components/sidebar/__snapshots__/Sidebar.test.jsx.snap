// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Sidebar renders itself 1`] = `
<div
  className="sidebar"
>
  <ul>
    <li>
      <a
        href="namespace"
      >
        Namespaces
      </a>
    </li>
    <li>
      <a
        href="pod"
      >
        Pods
      </a>
    </li>
  </ul>
</div>
`;
